dashboard/app/models/levels/level.rb

```diff
@@ -268,25 +268,44 @@ def delete_custom_level_file
   end
 
   TYPES_WITHOUT_IDEAL_LEVEL_SOURCE = [
-    'Unplugged', # no solutions
-    'TextMatch', # dsl defined, covered in dsl
-    'Multi', # dsl defined, covered in dsl
-    'External', # dsl defined, covered in dsl
-    'Match', # dsl defined, covered in dsl
-    'ContractMatch', # dsl defined, covered in dsl
-    'LevelGroup', # dsl defined, covered in dsl
     'Applab', # freeplay
-    'Gamelab', # freeplay
+    'ContractMatch', # dsl defined, covered in dsl
+    'CurriculumReference', # no user submitted content
+    'EvaluationMulti', # unknown
     'EvaluationQuestion', # plc evaluation
+    'External', # dsl defined, covered in dsl
+    'ExternalLink', # no user submitted content
+    'FreeResponse', # no ideal solution
+    'FrequencyAnalysis', # widget
+    'Gamelab', # freeplay
+    'GoBeyond', # unknown
+    'LevelGroup', # dsl defined, covered in dsl
+    'Map', # no user submitted content
+    'Match', # dsl defined, covered in dsl
+    'Multi', # dsl defined, covered in dsl
     'NetSim', # widget
+    'Pixelation', # widget
+    'PublicKeyCryptography', # widget
     'Odometer', # widget
-    'Vigenere', # widget
-    'FrequencyAnalysis', # widget
+    'ScriptCompletion', # unknown
+    'StandaloneVideo', # no user submitted content
     'TextCompression', # widget
-    'Pixelation', # widget
-    'PublicKeyCryptography' # widget
+    'TextMatch', # dsl defined, covered in dsl
+    'Unplugged', # no solutions
+    'Vigenere', # widget
+    'Weblab', # no ideal solution
   ].freeze
-  # level types with ILS: ["Craft", "Studio", "Karel", "Eval", "Maze", "Calc", "Blockly", "StudioEC", "Artist"]
+  # TYPES_WITH_IDEAL_LEVEL_SOURCE = [
+  #   'Artist',
+  #   'Blockly',
+  #   'Calc',
+  #   'Craft',
+  #   'Eval',
+  #   'Karel',
+  #   'Maze',
+  #   'Studio',
+  #   'StudioEC',
+  # ]
```

Comment:
Would love to see that check added if this is important, rather than depending on people realizing this is somewhere they need to look/something they need to think about.